### PR TITLE
Enhance and fix SSH workflows

### PR DESCRIPTION
**Pull request by nebilbenmabrouk**

### CODE REVIEW

File: SSH/resources/catalog/SSH_Service.xml [hunk 1]
```diff
@@ -13,11 +13,12 @@
     <variable name="NODE_SOURCE_NAME" value=""/>
   </variables>
   <description>
-    <![CDATA[ Deploy a SSH terminal on web browser using the Wetty tool. This service takes as input:
+    <![CDATA[ Deploy a SSH terminal on web browser using the Wetty tool. This service requires as input:
 - INSTANCE_NAME: name of SSH service instance
 - TARGET_HOST: SSH target host, among the list of hosts managed by ProActive Resource Manager
 - SSH_PORT: port used by SSH in the TARGET_HOST
-- PROXYFIED: whether the service should be accessed via our proxy (for restricted access control) ]]>
+- PROXYFIED: whether the service should be accessed via our proxy (for restricted access control)
```
Comment (tao-xinxiu) [inline SSH/resources/catalog/SSH_Service.xml:20]: "Our" could be more specific here. Does it mean the proxy provided by PA scheduler ?

**Review by tao-xinxiu — APPROVED** [review]

File: SSH/resources/catalog/SSH_Service.xml [hunk 2]
```diff
@@ -13,11 +13,12 @@
...
   </variables>
   <description>
-    <![CDATA[ Deploy a SSH terminal on web browser using the Wetty tool. This service takes as input:
+    <![CDATA[ Deploy a SSH terminal on web browser using the Wetty tool. This service requires as input:
 - INSTANCE_NAME: name of SSH service instance
 - TARGET_HOST: SSH target host, among the list of hosts managed by ProActive Resource Manager
 - SSH_PORT: port used by SSH in the TARGET_HOST
-- PROXYFIED: whether the service should be accessed via our proxy (for restricted access control) ]]>
+- PROXYFIED: whether the service should be accessed via our proxy (for restricted access control) 
+The service takes further inputs as variable values when used with a native scheduler such as Slurm. In this case, it deploys a SSH terminal and reserves one or several hosts allocated by the underlying native scheduler.]]>
```
Comment (amouhoub) [inline SSH/resources/catalog/SSH_Service.xml:21]: additional inputs

File: SSH/resources/catalog/SSH_Service.xml (hunk 1 above)
Comment (fviale) [reply SSH/resources/catalog/SSH_Service.xml:20]: via the ProActive server proxy

**Review by fviale — APPROVED** [review]
LGTM

File: SSH/resources/catalog/SSH_Terminal_Via_Browser.xml [hunk 3]
```diff
@@ -13,11 +13,12 @@
...
   </variables>
   <description>
-    <![CDATA[ Deploy a Wetty service to provide to users a SSH terminal via a web browser. This service takes as input:
+    <![CDATA[ Deploy a Wetty service to provide to users a SSH terminal via a web browser. This service requires as input:
 - INSTANCE_NAME: name of SSH service instance
 - TARGET_HOST: SSH target host, among the list of hosts managed by ProActive Resource Manager
 - SSH_PORT: port used by SSH in the TARGET_HOST
-- PROXYFIED: whether the service should be accessed via our proxy (for restricted access control) ]]>
+- PROXYFIED: whether the service should be accessed via our proxy (for restricted access control)
+The service takes further inputs as variable values when used with a native scheduler such as Slurm. In this case, it deploys a SSH terminal and reserves one or several hosts allocated by the underlying native scheduler.]]>
```
Comment (amouhoub) [inline SSH/resources/catalog/SSH_Terminal_Via_Browser.xml:21]: ... additional inputs ...

**Review by amouhoub — APPROVED** [review]
minor comments

File: SSH/resources/catalog/SSH_Service.xml (hunk 1 above)
Comment (nebilbenmabrouk) [reply SSH/resources/catalog/SSH_Service.xml]: Done. Thanks @tao-xinxiu @fviale

File: SSH/resources/catalog/SSH_Service.xml (hunk 2 above)
Comment (nebilbenmabrouk) [reply SSH/resources/catalog/SSH_Service.xml:21]: This sentence was validated by Denis. I prefer to keep it like that :)

**Pull request merged**